yarn.lock

```diff
@@ -146,7 +146,7 @@
   "resolved" "https://registry.npmjs.org/@babel/helper-compilation-targets/-/helper-compilation-targets-7.13.13.tgz"
   "version" "7.13.13"
   dependencies:
-    "@babel/compat-data" "^7.13.12"
+    "@babel/compat-data" "^7.13.15"
```

Comment:
same with these changes. We should probably have an OS-specific switch that determines the correct versions here.